packages/client-python/src/rocketride/mixins/store.py

```
@@ -21,868 +21,334 @@
-        return response_body
+    # =========================================================================
+    # Domain Convenience - Projects
+    # =========================================================================
 
-    async def get_all_templates(self) -> Dict[str, Any]:
-        """
-        List all templates (system-wide).
+    async def save_project(self, project_id: str, pipeline: Dict[str, Any]) -> None:
+        """Save a project pipeline to .projects/<project_id>.json."""
+        if not project_id:
+            raise ValueError('project_id is required')
```

Ruff (0.15.7)

[warning] Avoid specifying long messages outside the exception class

(TRY003)